### DISCUSSION TITLE
Can i bypass translateZ property in cube effect?

### DISCUSSION THREAD

**Discussion by bilalmalkoc** [nolimits4web/swiper#5114] [Q&A]
Cube effect adding transform style for swaper-wrapper class. But it causes some bugs for ios. I think i can fix it by set translateZ value to zero. I mentioned here https://github.com/nolimits4web/swiper/issues/5081

Change this `transform: translate3d(0px, 0px, -187.5px) rotateX(0deg) rotateY(0deg);`

To this

`transform: translate3d(0px, 0px, 0) rotateX(0deg) rotateY(0deg);`

Will this be useful? Or maybe there is a better way to fix that.

**Comment by nolimits4web** [comment]
If you remove translateZ then "Cube" will be like 1.5x times enlarged, due to same Safari bug

**Reply by bilalmalkoc** [reply]
> If you remove translateZ then "Cube" will be like 1.5x times enlarged, due to same Safari bug

Thanks. Do you planning fix this?

**Comment by bilalmalkoc** [comment]
I really need a solution for this problem :) any good news about this?

**Comment by nolimits4web** [comment]
It is issue with Safari, not in Swiper
